tutorials/cloud-functions-sns-pubsub/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Connect Google Cloud Pub/Sub to AWS SNS topics through Cloud Functions
+description: Learn how to integrate Google Cloud Pub/Sub with AWS SNS using Google Cloud Functions.
+author: ptone
+tags: Cloud Functions, AWS, SNS, Node.js
+date_published: 2017-06-23
+---
+## Introduction
+
+This tutorial and sample function demonstrates using [Google Cloud
+Functions][functions] as an integration point between the [Amazon Simple
+Notification Service][sns] (SNS) and Google Cloud Pub/Sub (Cloud Pub/Sub).  The
+function is implemented in [Node.js][node].
+
+[functions]: https://cloud.google.com/functions
+[twilio]: https://www.twilio.com/
+[node]: https://nodejs.org/en/
+[sns]: https://aws.amazon.com/sns/
+
+The SNS service makes a POST request to the Cloud Function URL when a
+message is published to the corresponding SNS topic. The function validates the
+sender and topic.
+
+The function then publishes the message to Cloud Pub/Sub - tagging the message
+with attributes of SNS subject, and message id.
+
+## Prerequisites
+
+1.  Create a project in the [Google Cloud Platform Console][console].
+1.  [Enable billing][billing] for your project.
+1.  Install the [Google Cloud SDK][sdk].
+1.  Have an AWS console account with access to the SNS service.
+
+[console]: https://console.cloud.google.com/
+[sdk]: https://cloud.google.com/sdk/
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+
+
+## Setting up the SNS topic
+
+For this section it is assumed that you are already familiar with AWS SNS,
+[create an SNS topic for this tutorial][sns-create] if you do not already have
```

Comment:
Change this: already familiar with AWS SNS, [create an SNS topic for this tutorial][sns-create] if

To this: already familiar with AWS SNS.  [Create an SNS topic for this tutorial][sns-create] if